test-requirements.txt

```diff
@@ -12,7 +12,7 @@ coverage-config-reload-plugin~=0.2
 codecov~=2.0.5
 moban~=0.3.3 ; python_version > '3.0'
 packaging~=16.8
-pytest~=3.6.1
+pytest~=4.4.0
```

Comment:
this needs to be the same as the version specified by coala mobans , which should be ok to increase because I've fixed the coverage plugins

Reply:
Yes! It was generated by moban sync. Doing suggested changes will make it same as specified by coala mobans 😇

Reply:
Yes, but then you are also proposing to change the version in the template at the mobans repo, which would mean it needs to change here again.  We need one version everywhere, so we need to find one set of versions which work everywhere.